config/webpack.config.dev.js

```diff
@@ -25,9 +25,7 @@ module.exports = {
     new webpack.DefinePlugin({
       __STACK_ASSETS__: false,
       __DEV__: true,
-      'process.env': {
-        NODE_ENV: JSON.stringify('development')
-      }
+      'process.env.NODE_ENV': JSON.stringify('development')
```

Comment:
I dont understand well the difference ? What is the purpose ?

Reply:
The previous form was the same as doing

```js
process.env = { NODE_ENV: 'development' }
```

We lost all other `process.env` properties (including `COZY_*` provided by the stack). Now we just provide the `NODE_ENV` property without touching other ones.

Reply:
Nice catch, thanks for the explanation.